modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+|`ami-02c056bb061ecada0`
+|`rhcos-415.92.202309201557-0-x86_64`
+
+.6+|*eu-south-2*
+|`ami-0e7c6506841f137c0`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0b8953c88a0645080`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-031e602fbf1087ed3`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-06838fa4e8b987cd8`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0f3ad9b25036a95dc`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0880fa6c19e3d426a`
+|`rhcos-411.85.202203242008-0-x86_64`
+
+.19+|*eu-north-1*
+|`ami-09c872e8081e9d63d`
+|`rhcos-45.81.202005200134-0`
+|`ami-080f460264d851577`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-0f8142237d51621db`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-099f3930c1ec7cc46`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-02fbec4b51b9c4793`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0435b2118cccddb6b`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-076637160d6b221ff`
+|`rhcos-45.82.202008010929-0`
+|`ami-07d141e3e87a9dc80`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0e8f22283e46e8cb3`
+|`rhcos-46.82.202008181646-0`
+|`ami-00be34c2341c4d4e2`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-04bc6ffc11b85dcc9`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-0e84383413e10ca0c`
+|`rhcos-47.83.202102090044-0`
+|`ami-0310e8f4a9fe48b28`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0bbdaf5e44a35ae8e`
+|`rhcos-49.84.202107191624-0`
+|`ami-0194bfc5560eb87b3`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0936d778bd30afd2c`
+|`rhcos-48.84.202105062123-0`
+|`ami-0702ddb20f5c83d22`
+|`rhcos-48.84.202107202156-0`
+|`ami-06cf364d11a3b4460`
+|`rhcos-46.82.202008071940-0`
+|`ami-0fb33a85b51cb99ef`
+|`rhcos-414.92.202305090606-0-x86_64`
+
+.6+|*eu-central-2*
+|`ami-03a91f2453b29da56`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0b427ee1b18f44703`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-0f16a11c6c6fd1bff`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-091ff7e0f64c6b5c1`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-0060d33ef19ed919a`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-04c6ff1edabf3f212`
+|`rhcos-414.92.202305090606-0-x86_64`
+
+.19+|*me-south-1*
+|`ami-0390b2b0e4d2e9f29`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-085da42db9a56d179`
+|`rhcos-48.84.202107202156-0`
+|`ami-0c8fe68690466f8fb`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-01585e03454ef2331`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-017183981c852d8a9`
+|`rhcos-45.81.202005200134-0`
+|`ami-009342b54bd186748`
+|`rhcos-46.82.202008181646-0`
+|`ami-0b1188c3924a8fbef`
+|`rhcos-47.83.202102090044-0`
+|`ami-02ac05d07d8367b93`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-072798834f514e07d`
+|`rhcos-45.82.202008010929-0`
+|`ami-0375dbe58929d3597`
+|`rhcos-46.82.202008071940-0`
+|`ami-02d91107af59d29a2`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-04d7d9268a0547bef`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-09fdee1c36505b3d1`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?